src/main/java/bwapi/GameDataUtils.java

```diff
@@ -0,0 +1,79 @@
+/*

```

Comment:
Don't need to put the LICENSE here, only one file from Hannes was used, which was licensed under MIT, which is still there for those reasons, for the other files the repo wide MIT license should be valid.

If not, I'll automate to add to all files later, but that doesn't have to be done for this PR

Reply:
I think I included it here because this code was copy-pasted from that file. Will double check.

Reply:
Confirmed: this came from https://github.com/JavaBWAPI/JBWAPI/blob/develop/src/main/java/bwapi/Client.java#L267 which is under Hannes' license.